site/docs/v1/tech/connectors/index.adoc

```diff
@@ -35,12 +35,34 @@ In contrast to using an OIDC Identity Provider, with a Connector the user doesn'
 
 Using a Connector for authentication allows you to use an external user management system as another data source for FusionAuth. If the data source can speak LDAP or HTTP, you can authenticate users against it.
 
+==== Required fields
+
+If you are authenticating a user, you must provide the following fields in the user object you return.
+
+* `user.username` or `user.email`
+* `user.id`: a FusionAuth compatible GUID
```

Comment:
We don't use the term `GUID` anywhere else in FusionAuth, we use `UUID`. 

> FusionAuth compatible UUID.

Reply:
Awesome, thanks. Fixed.